stl/inc/random

```diff
@@ -179,6 +179,11 @@ public:
 
     template <class _RanIt>
     void generate(_RanIt _First, _RanIt _Last) const { // generate randomized interval from seeds
+        static_assert(_Is_any_of_v<remove_cv_t<typename iterator_traits<_RanIt>::value_type>, //
+                          unsigned int, unsigned long, unsigned long long>,
+            "N4971 [rand.util.seedseq]/7 requires the value type of the iterator to be an unsigned type that is at "
+            "least 32-bit.");

```

Comment:
No change requested: I wouldn't have used `remove_cv_t` here (elsewhere we assume that `iterator_traits::value_type` is cv-unqualified), and @CaseyCarter would have said "unsigned integer type" to match the Standard, but neither are worth resetting testing.

Reply:
Hmm... [[fs.path.req]/2.2](https://eel.is/c++draft/fs.path.req#2.2) currently accepts a const-qualified `value_type`, but [[fs.req]/3](https://eel.is/c++draft/fs.req#3) doesn't seem so. Perhaps there should be an LWG issue for this.